microcosm_pubsub/sentry.py

```diff
@@ -33,41 +53,80 @@ def before_send(event, hint):
                 if var_name in ["self", "cls"]:
                     continue
                 elif isinstance(value, str):
-                    if value.startswith("<function "):
+                    if value.startswith("<") and value.endswith(">"):
                         continue
                     elif var_name.endswith("_id"):
                         continue
+                    elif value.startswith("http"):
+                        continue
+                elif isinstance(value, dict):
+                    for key in value.keys():
+                        if key.endswith("_id"):
+                            continue
+                        value[key] = "<redacted>"
+                    continue
                 frame["vars"][var_name] = "<redacted>"
     return event
 
 
 @defaults(
     enabled=False,
     dsn=None,
+    custom_tags_mapping=None,
+    custom_before_send_func=None,
+    custom_user_id=None,
 )
 @logger
-def configure_sentry(graph: ObjectGraph):
-    enabled, dsn, sentry = False, None, None
+def configure_sentry(graph: ObjectGraph) -> SentryConfig:
+    enabled, dsn, sentry,  = False, None, None
```

Comment:
Nit: inconsistent use of trailing commas (and a bonus space).

Reply:
🦅 👁️ @simpleigh-globality 👍